src/System.CommandLine.DragonFruit.Tests/ConfigureFromMethodTests.cs

```diff
@@ -221,6 +221,19 @@ public void Options_are_not_built_for_infrastructure_types_exposed_by_method_par
                    .NotContain(o => o.Argument.ArgumentType == type);
         }
 
+        [Fact]
+        public void Parameters_with_Argument_suffix_is_treated_as_arguments()
```

Comment:
This is great, but I would also include tests for when the argument names end with "arguments" and "args"

Reply:
The thing here that will need more work is that the method parameter name will be, e.g., `fooArg` but the `Argument.Name` (as reflected in help) would be `foo`. This means binding will need customized behavior as well.

Thinking this through led me back to the possibility of introducing attributes, and then perhaps solving the alias problem (#525) at the same time. I've been reluctant to do this before but maybe it's a the more evolvable approach.

Reply:
Can we just have one suffix (‘Argument’ or ‘Arg’)?

Reply:
I think the question first is whether we would need both a naming convention and an attributed approach. They would be redundant within one app model. This really would expand DragonFruit into a new direction, which might be good, but would certainly become more complex.

Reply:
Both are good, right? Just like asp.net ... you can use Attribute or Convention.

Reply:
Yes, but a lot more complexity.

Reply:
I personally prefer By Convention.